src/ParallelHashMap.h

```diff
@@ -244,7 +244,7 @@ void FixedHashMap<K,V>::insert(K key, V value)
   /* check to see if key already exists in map */
   if (contains(key))
     return;
-
+ 
```

Comment:
I think they are due to @geogunow not having something in his vi settings file to remove all extra whitespace when the file is saved like I do for emacs. Github doesn't do show whitespace added on new lines so it is kind of hard to catch this on new edits, unless the new edit is just adding whitespace to a blank line.